test/functional/wallet_basic.py

```diff
@@ -136,11 +136,20 @@ def run_test(self):
                                 self.nodes[2].lockunspent, False,
                                 [{"txid": unspent_0["txid"], "vout": 999}])
 
-        # An output should be unlocked when spent
+        # The lock on a manually selected output is honoured
         unspent_0 = self.nodes[1].listunspent()[0]
         self.nodes[1].lockunspent(False, [unspent_0])
         tx = self.nodes[1].createrawtransaction([unspent_0], { self.nodes[1].getnewaddress() : 1 })
-        tx = self.nodes[1].fundrawtransaction(tx)['hex']
```

Comment:
Are you removing this case because it's used elsewhere?

Reply:
The original test was for auto-unlocking a coin, which now produces an error, so I check for that.